test/apm_repo.js

```diff
@@ -1,6 +1,7 @@
 const { assertRevert } = require('./helpers/assertThrow')
 
 const Repo = artifacts.require('Repo')
+const getContract = artifacts.require
```

Comment:
Seems unnecessary?

Reply:
In order to support copying code from migrations lol